### PR TITLE
rad

### PR DESCRIPTION
**Pull request by rstorr**

### CODE REVIEW

File: a.go [hunk 1]
```diff
@@ -7,3 +7,7 @@ import (
 func aFunc() {
 	fmt.Println("the following sentence is true. the preceding sentence is false.")
 }
+
+func rad() {
+	fmt.Println("the following sentence is true. the preceding sentence is false.")
```
Comment (rstorr) [inline a.go:12]: @stagingcodelingobot capture

Comment (rstorr) [reply a.go:12]: @stagingcodelingobot capture

Comment (waigani) [reply a.go:12]: @stagingcodelingobot capture

Comment (rstorr) [reply a.go:12]: @stagingcodelingobot capture

Comment (Daanikus) [reply a.go:12]: @stagingcodelingobot capture

**Comment by Daanikus** [comment]
@stagingcodelingobot capture

File: a.go [hunk 2]
```diff
@@ -7,3 +7,7 @@ import (
 func aFunc() {
 	fmt.Println("the following sentence is true. the preceding sentence is false.")
 }
+
+func rad() {
```
Comment (Daanikus) [inline a.go:11]: @stagingcodelingobot capture

Comment (rstorr) [inline a.go:11]: baddd practice here...

Comment (rstorr) [reply a.go:11]: @CodeLingoBot capture

File: a.go [hunk 3]
```diff
@@ -7,3 +7,7 @@ import (
 func aFunc() {
 	fmt.Println("the following sentence is true. the preceding sentence is false.")
 }
+
```
Comment (waigani) [inline a.go:10]: my comment

Comment (waigani) [reply a.go:10]: @CodeLingoBot capture

Comment (Daanikus) [reply a.go:10]: @CodeLingoBot capture

Comment (rstorr) [reply a.go:10]: @stagingcodelingobot capture

File: a.go (hunk 1 above)
Comment (fwereade) [reply a.go:12]: @stagingcodelingobot capture

File: a.go (hunk 2 above)
Comment (fwereade) [reply a.go:11]: @stagingcodelingobot capture this is a rule with a title

**Pull request closed without merging**